packages/signer/src/app/screens/Accounts.tsx

```diff
@@ -82,7 +85,26 @@ export const Accounts = () => {
     return () => dispatch(setNavigationBack(undefined));
   }, []);
 
-  return (
+  return accounts.length === 0 ? (
+    <>
+      <Container pt="4">
+        <Box sx={{ textAlign: 'center' }}>
+          <Image alt="Wallet" src={wallet} />
+        </Box>
+        <Box maxWidth="80%" mx="auto" sx={{ textAlign: 'center' }}>
+          <Heading fontSize="24px" lineHeight="150%" mt="3" mb="2">
+            {translateRaw('ACCOUNTS_EMPTY_HEADER')}
+          </Heading>
+          <Body variant="muted">{translateRaw('ACCOUNTS_EMPTY_BODY')}</Body>
+        </Box>
+      </Container>
+      <PanelBottom variant="clear">
+        <LinkApp href={ROUTE_PATHS.ADD_ACCOUNT}>
+          <Button>{translateRaw('MENU_ADD_ACCOUNT')}</Button>
+        </LinkApp>
+      </PanelBottom>
+    </>
+  ) : (
```

Comment:
Maybe do something like this to make it slightly more readable?
```ts
if (accounts.length === 0) {
  return ...;
}

return ...;
```

Reply:
Sure